tests/subyt/out/01-basic.ttl

```diff
@@ -16,8 +16,8 @@
   schema:description '''een tekst over
 meerdere \'\'\'\'\'
 lijnen'''@nl;
-  schema:description '''ceci n'est pas une texte'''@fr;
-  schema:description 'As \\ said before'@en;
+  schema:description '''ceci n\'est pas une texte'''@fr;
+  schema:description 'As \ said before'@en;
```

Comment:
but very unsure why this changed? 
single \ should become \\ indeed - as was originally in the test?